cartographer/mapping_3d/sparse_pose_graph/constraint_builder.cc

```diff
@@ -179,12 +179,16 @@ void ConstraintBuilder::ComputeConstraint(
   const sensor::PointCloud filtered_point_cloud =
       adaptive_voxel_filter_.Filter(point_cloud);
 
-  // The 'constraint_transform' (submap 'i' <- scan 'j') is computed from the
-  // initial guess 'initial_pose' for (submap 'i' <- scan 'j') and a
-  // 'filtered_point_cloud' in 'j'.
+  // The 'constraint_transform' (submap i <- scan j) is computed from:
```

Comment:
I made this also more similar to the formulation in 2D. However please double check since i'm not very familiar with the 3D case.